packages/core/src/components/image-overlay/image-overlay.tsx

```diff
@@ -11,6 +18,12 @@ export class ImageOverlay implements ComponentInterface {
   @Element()
   host!: HTMLDsoImageOverlayElement;
 
+  /**
+   * The wijzigactie as in STOP.
+   */
+  @Prop()
+  wijzigactie?: string | null;
```

Comment:
Vanwaar ook `| null` ?

Reply:
Ik heb deze property toegevoegd om de `wijzigactie` binnen `dso-image-overlay` beschikbaar te hebben om `ins` en `del` aldaar te kunnen renderen, maar tegelijkertijd is het niet bedoeld als een public interface van het web-component `dso-image-overlay`, maar 'slechts' gekoppeld aan de ozon figuur node ... dus alleen bedoeld als "internal" of "hidden" property. Is dat dan op een andere manier te organiseren?

Dus dit is een beetje krom.

Idealiter willen we die renvooi weergave functionaliteit buiten dit component houden, echter het vergoten van de afbeelding zit binnen dso-image-overlay, dus was dit nodig. Of zie jij @tfrijsewijk hier nog een andere mogelijkheid?

Reply:
Ik vraag mij met name af of het gedrag van `undefined` en `null` hetzelfde is. Zo ja: Samenvoegen tot 1 type, zo nee: documenteren

Reply:
Heb ik aangepast